packages/tasks/src/model-libraries.ts

```diff
@@ -666,6 +666,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: true,
 		countDownloads: `path:"models/default.zip"`,
 	},
+	"t5-tts": {
+		prettyLabel: "T5-TTS",
+		repoName: "T5-TTS",
```

Comment:
```suggestion
	"f5-tts": {
		prettyLabel: "F5-TTS",
		repoName: "F5-TTS",
```

Looks like the repo is called "F5" instead of "T5".
See also https://huggingface.co/models?other=f5-tts.